lib/improver/tests/nbhood/square_kernel/test_SquareNeighbourhood.py

```diff
@@ -1087,9 +1110,9 @@ def test_multiple_times_nan(self):
              [1., 0.88888889, 0.88888889, 0.88888889, 1.],
              [1., 1., 1., 1., 1.]])
         expected_2 = np.array(
-            [[0.88888889, 0.88888889, 0.88888889, 1., 1.],
-             [0.88888889, np.nan, 0.88888889, 1., 1.],
-             [0.88888889, 0.88888889, 0.88888889, 1., 1.],
+            [[1., 1., 1., 1., 1.],
```

Comment:
I'm not sure why these values have changed.

Reply:
The only values in the original array were 1.0 so min/max values were 1.0 hence after clipping the only values available are 1.0. The np.nan shouldn't be reducing the values in the neighbourhood. This is a bug that will be fixed in the rewrite version.